CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
-* None.
+* An exception is thrown when a realm file has invalid top ref.
```

Comment:
```suggestion
* Instead of asserting, an ??? exception is thrown when a realm file is opened with an invalid top ref.
```

Reply:
A new commit with this change has been made